pennylane/ops/qubit/state_preparation.py

```diff
@@ -176,6 +171,34 @@ def compute_decomposition(state, wires):
         """
         return [MottonenStatePreparation(state, wires)]
 
+    def state_vector(self, wire_order=None):
+        num_op_wires = len(self.wires)
+        op_vector = reshape(self.parameters[0], (2,) * num_op_wires)
+
+        if wire_order is None:
+            return op_vector
+
+        wires = Wires(wire_order)
```

Comment:
Any reason not to keep the term `wire_order`? I think `wire_order` helps distinguish it a bit from "wires" being the operation `self.wires`.

Reply:
it's a slight difference but not a problem, i'll update accordingly!

Reply:
on second thought, I only need it to be a `Wires` instance for this check, so I'll just cast it there